kubeflow/core/jupyterhub_spawner.py

```diff
@@ -100,7 +100,7 @@ def extra_resource_limits(self):
 # TODO(jlewi): Should we set c.KubeSpawner.singleuser_fs_gid = 1000
 # see https://github.com/kubeflow/kubeflow/pull/22#issuecomment-350500944
 pvc_mount = os.environ.get('NOTEBOOK_PVC_MOUNT')
-if pvc_mount:
+if pvc_mount and pvc_mount != 'null':
```

Comment:
This is good for now but it would be better to do this in the jsonnet rather than propogating the hack of 'null' to.

Reply:
yeah, but I can't find out how to assign empty string to default. As soon as it's fixed in ksonnet, we should do that

Reply:
I meant in ksonnet we should continue to use "null" and then in ksonnet we should have a utility function here
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/core/util.libsonnet#L14

Then can automatically handle converting "null" to ""